### PR TITLE
Disconnect to lobby

### PR DESCRIPTION
**Pull request by james58899**

### CODE REVIEW

**Comment by james58899** [comment]
Waiting upstream api change merge

**Comment by jimchen5209** [comment]
- [ ] #20 Merged

File: src/main/kotlin/one/oktw/galaxy/proxy/Main.kt [hunk 1]
```diff
@@ -114,6 +116,13 @@ class Main {
                 it.result = ServerPreConnectEvent.ServerResult.allowed(lobby)
             }
 
+            // Connect back to lobby on disconnect from galaxies
+            proxy.eventManager.register(this, KickedFromServerEvent::class.java) {
+                if (it.server == lobby || !this::lobby.isInitialized || it.kickedDuringServerConnect()) return@register // Ignore exist player
+
+                it.result = KickedFromServerEvent.RedirectPlayer.create(lobby, TextComponent.empty())
```
Comment (jimchen5209) [inline src/main/kotlin/one/oktw/galaxy/proxy/Main.kt:123]: Empty?

Comment (james58899) [reply src/main/kotlin/one/oktw/galaxy/proxy/Main.kt:123]: CC @bill96012

**Pull request merged**